docker/Jenkinsfile-build-docker

```diff
@@ -20,9 +20,9 @@ properties([
 IMAGE_NAME_PREFIX = 'harbor.h2o.ai/opsh2oai/h2o-3'
 
 JDK_VERSIONS = ['8', '10', '11', '12', '13', '14', '15']
-JDK_VERSIONS_PY_R = ['8', '11', '14', '15'] // stable, last-supported, latest
-PYTHON_VERSIONS = ['2.7', '3.5', '3.6', '3.7', '3.8']
-R_VERSION = ['3.3.3', '3.4.1', '3.5.3', '3.6.2', '4.0.2']
+JDK_VERSIONS_PY_R = ['8', '11', '14', '15'] // first_LTS, latest_LTS, latest (see scripts/jenkins/groovy/buildConfig.groovy)
+PYTHON_VERSIONS = ['2.7', '3.5', '3.7', '3.8'] // legacy, first supported Py3, first active Py3, latest (see scripts/jenkins/groovy/buildConfig.groovy)
+R_VERSIONS = ['3.3.3', '3.4.4', '3.5.3', '3.6.3', '4.0.5', '4.1.2'] // do we really need to build an image for all those versions given that we test only one v3 + one v4?
```

Comment:
I feel like reducing the list of R versions to test: not sure it's worth testing all minor versions since 3.3 + there are now 2 minor 4.x versions available